scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -2132,6 +2009,116 @@
                 "schedule each pool less often than the desired setting of every " target-per-pool-match-interval-millis " ms."))
     (async/pipe (chime-ch (util/time-seq (time/now) (time/millis match-interval-millis))) match-trigger-chan)))
 
+(defn handle-fenzo-pool
```

Comment:
This should be identical to `make-offer-handler` except for some variables defined in the caller such as `user->quota`.

Reply:
Can you run the diff for what changed. (FYI: kdiff3 lets you do manual alignment.) One slight concern is changing the order of when we fetch from the DB can made race windows wider or narrower. That may be fine, but I want such changes to be intentional.

Reply:
Alternate approach. Move the implementation of handle-fenzo-pool up to where make-offer-handler used to be, that way the diff is much smaller and github will do the right thing.